src/sick_tim_common_usb.cpp

```diff
@@ -265,6 +265,7 @@ int SickTimCommonUsb::sendSOPASCommand(const char* request, std::vector<unsigned
   if (device_handle_ == NULL) {
     ROS_ERROR("LIBUSB - device not open");
     diagnostics_.broadcast(diagnostic_msgs::DiagnosticStatus::ERROR, "LIBUSB - device not open.");
+    sleep(1);
```

Comment:
I would propose to move this into the outer `while (ros::ok())` and use a `ros::Rate(1);`. Rational: It's more visible and can be adopted easily later. The other option would be `SickTimCommonUsb::init_device` btw.